### PR TITLE
Julia Kozeeva Task 2 https://julia-kozeeva-task2.firebaseapp.com/

### PR DESCRIPTION
**Pull request by juliakozeeva**

### CODE REVIEW

File: task2/public/style/sass/components/_burger-menu.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+  &__label {
+    display: block;
+    position: relative;
+    height: inherit;
+    width: inherit;
+    z-index: 11;
+
+    &:hover {
+      cursor: pointer;
```
Comment (savchukm) [inline task2/public/style/sass/components/_burger-menu.scss]: Please add cursor pointer to the element itself

File: task2/public/style/sass/components/_burger-menu.scss [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+  &__line {
+    position: absolute;
+    width: 100%;
+    height: 2px;
+    top: 50%;
+    margin-top: -1px;
+    left: 0;
+    display: block;
+    background-color: #020304;
```
Comment (savchukm) [inline task2/public/style/sass/components/_burger-menu.scss]: I'd recommend using sass variables holding any color what you have

File: task2/public/style/sass/layout/grid.scss [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .container {
+    margin: 66px;
+  }
+
+  .footer {
+    padding: 40px !important;
```
Comment (savchukm) [inline task2/public/style/sass/layout/grid.scss]: !important is bad practice. Can you try overspecify selector to apply padding?

File: task2/private/sass/components/_burger-menu.scss [hunk 4]
```diff
@@ -0,0 +1,118 @@
+@import '../abstracts/colors';
```
Comment (vadeneev) [inline task2/private/sass/components/_burger-menu.scss]: I assume, it is not abstracts, but variables ?

File: task2/private/sass/components/_burger-menu.scss [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+  &__line {
+    position: absolute;
+    width: 100%;
+    height: 2px;
+    top: 50%;
+    margin-top: -1px;
+    left: 0;
+    display: block;
+    background-color: $span-color;
```
Comment (vadeneev) [inline task2/private/sass/components/_burger-menu.scss]: That what I wrote - background uses span-color ... what ?

File: task2/private/sass/components/_burger-menu.scss [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+    }
+
+    .burger-menu__line:last-child {
+      top: 50%;
+      opacity: 1;
+      transform: rotate(-140deg);
+    }
+  }
+
+  .mobile-navbar {
```
Comment (vadeneev) [inline task2/private/sass/components/_burger-menu.scss]: it worth to move this block to its own file

File: task2/private/sass/layout/grid.scss [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+      padding: 0;
+    }
+  }
+  section.mail-container{
+    padding: 50px 30px;
+  }
+}
+
+@media screen and (max-width: 500px) {
+  nav.navbar {
```
Comment (vadeneev) [inline task2/private/sass/layout/grid.scss]: wait, no, it is not about BEM

File: task2/private/sass/layout/grid.scss [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+}
+.wrapper {
+  flex: 1 0 auto;
+}
+
+.container-fixed-width {
+  @include fixed-with;
+}
+
+.container {
```
Comment (vadeneev) [inline task2/private/sass/layout/grid.scss]: Try to keep each block in separate file, all atomic styles in own _common or _general

File: task2/private/sass/layout/grid.scss [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+    margin: 66px 80px;
+  }
+}
+
+@media screen and (max-width: 1024px) {
+  .container {
+    margin: 66px;
+  }
+
+  footer.footer {
```
Comment (vadeneev) [inline task2/private/sass/layout/grid.scss]: 🚫 🙅‍♂

File: task2/private/sass/layout/grid.scss [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+    flex-direction: column;
+    &__img {
+      height: auto;
+    }
+
+    &-content {
+      padding: 0;
+    }
+  }
+  section.mail-container{
```
Comment (vadeneev) [inline task2/private/sass/layout/grid.scss]: 🚫 🙅‍♂

File: task2/private/sass/layout/grid.scss [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+    padding: 50px 30px;
+  }
+}
+
+@media screen and (max-width: 500px) {
+  nav.navbar {
+    padding: 30px;
+  }
+
+  section.img-container {
```
Comment (vadeneev) [inline task2/private/sass/layout/grid.scss]: 🚫 🙅‍♂

File: task2/private/sass/sections/_header.scss [hunk 12]
```diff
@@ -0,0 +1,56 @@
+.navbar {
```
Comment (vadeneev) [inline task2/private/sass/sections/_header.scss]: Try to name files in same way as Blocks - header <-> navbar ❓

File: task2/private/sass/sections/_img-row.scss [hunk 13]
```diff
@@ -0,0 +1,9 @@
+.img-container {
```
Comment (vadeneev) [inline task2/private/sass/sections/_img-row.scss]: ing-row <-> img-container ?
try to use same names

File: task2/public/assets/send_icon.svg [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 448.011 448.011" style="enable-background:new 0 0 448.011 448.011;" xml:space="preserve">
+<g>
+	<g>
+		<path d="M438.731,209.463l-416-192c-6.624-3.008-14.528-1.216-19.136,4.48c-4.64,5.696-4.8,13.792-0.384,19.648l136.8,182.4
+			l-136.8,182.4c-4.416,5.856-4.256,13.984,0.352,19.648c3.104,3.872,7.744,5.952,12.448,5.952c2.272,0,4.544-0.48,6.688-1.472
+			l416-192c5.696-2.624,9.312-8.288,9.312-14.528S444.395,212.087,438.731,209.463z"/>
+	</g>
+</g>
+<g>
```
Comment (vadeneev) [inline task2/public/assets/send_icon.svg]: it is all about space...
https://www.svgminify.com/

**Pull request merged**